fb.d.ts

```diff
@@ -0,0 +1,61 @@
+import Promise = require("any-promise");
```

Comment:
While that is true, that doesn't mean that everyone is using an ES6 environment, and additionally this library uses `any-promise` as the Promise implementation.